### PR TITLE
Set the get vertices/edges response columns name by plan. Change _vid…

### PR DESCRIPTION
**Pull request by Shylock-Hg**
… column to VertexID.

Change _vid column to VertexID.

### CODE REVIEW

**Review by CPWstatic — APPROVED** [review]

File: src/validator/GoValidator.cpp [hunk 1]
```diff
@@ -683,6 +684,25 @@ std::vector<storage::cpp2::VertexProp> GoValidator::buildDstVertexProps() {
...
 
+std::vector<std::string> GoValidator::buildDstVertexColNames() {
+    std::vector<std::string> dstColNames;
+    dstColNames.reserve(exprProps_.dstTagProps().size());
+    dstColNames.emplace_back(kVid);
+    for (const auto &tagProps : exprProps_.dstTagProps()) {
+        auto tagId = tagProps.first;
+        auto tagNameResult = qctx_->schemaMng()->toTagName(space_.id, tagId);
+        DCHECK(tagNameResult.ok());
+        if (!tagNameResult.ok()) {
```
Comment (laura-ding) [inline src/validator/GoValidator.cpp:695]: if `tagNameResult` is not ok, couldn't make validator continue. I think the tagName can be saved in exprProps_, so here not need to get tagName from tagId.

**Review by yixinglu — APPROVED** [review]
LGTM

File: src/validator/GoValidator.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/validator/GoValidator.cpp:695]: This had be checked out of this functions, for more independent we could return error.

Comment (Shylock-Hg) [reply src/validator/GoValidator.cpp:695]: But I don't think it's meanful.

Comment (Shylock-Hg) [reply src/validator/GoValidator.cpp]: Fixed.

**Review by laura-ding — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**